readme.md

```diff
@@ -158,7 +158,7 @@
 - [JVM](https://github.com/deephacks/awesome-jvm#readme)
 - [Salesforce](https://github.com/mailtoharshit/awesome-salesforce#readme)
 - [Amazon Web Services](https://github.com/donnemartin/awesome-aws#readme)
-- [Windows](https://github.com/Awesome-Windows/Awesome#readme)
+- [Windows](https://github.com/0pandadev/awesome-windows#readme) - The most used operating sytem for consumer PC's.
```

Comment:
(A suggestion, not a requirement or a part of the review.)

Let's maybe make it more up to the actual topic of the list, and not just a statement about what Windows is? Something like:
```suggestion
- [Windows](https://github.com/0pandadev/awesome-windows#readme) - Tools and apps for Windows 10/11.
```